components/cli/pkg/commands/instance/testdata/virtual-services/pet-fe-src-vs.yaml

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
Why we need to have a json virtual service? If its required, should change the extension to json.

Reply:
Updated to json.